packages/story-editor/src/components/canvas/layout.js

```diff
@@ -49,10 +49,12 @@ export const Z_INDEX = {
 };
 
 const HEADER_GAP = 16;
-const MENU_HEIGHT = THEME_CONSTANTS.ICON_SIZE;
+// 8px extra is for the focus outline to display.
+const MENU_HEIGHT = THEME_CONSTANTS.ICON_SIZE + 8;
```

Comment:
Doesn't that increase the gap between the page and the menu icons and similar between the menu icons and the carousel? Don't we need to reduce the gap on either side of the menu by 4 then?

Reply:
Changed.